index.html

```diff
@@ -14,14 +14,42 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Asap:wght@100;300;400;500;600&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
     <div class="container">
+      <h2 class="header-title">Code Journal</h2>
     </div>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <div data-view="entry-form">
+      <h1 class="new-entry">New Entry</h1>
+
+      <form class="entry-form">
+        <div class="img">
+          <img src="images/placeholder-image-square.jpg" alt="placeholder-image" class="placeholder-image">
+        </div>
+        <div class="input">
+          <label>Title</label>
+          <input type="text" class="title" required>
+        </div>
+        <div class="input">
+          <label>Photo URL</label>
+          <input type="url" class="photo" required>
+        </div>
+        <div class="input-notes">
+          <label class="notes-label">Notes</label>
+          <textarea class="notes" required></textarea>
+        </div>
+        <div class="button">
+          <button type="submit" class="save-button">SAVE</button>
+        </div>
+
+      </form>
```

Comment:
You should be using layout classes to handle the spacing of all your columns. Take a look at `layout.css` to see all the classes available to you. For instance, if you want two columns to be side by side, you should give them a class of `column-half`